cmd/nerdctl/container_run.go

```diff
@@ -184,6 +184,7 @@ func setCreateFlags(cmd *cobra.Command) {
 	cmd.Flags().StringSlice("cap-drop", []string{}, "Drop Linux capabilities")
 	cmd.RegisterFlagCompletionFunc("cap-drop", capShellComplete)
 	cmd.Flags().Bool("privileged", false, "Give extended privileges to this container")
+	cmd.Flags().String("systemd", "false", "Enable systemd integration (default: false)")
```

Comment:
This differs form the [podman flag](https://docs.podman.io/en/latest/markdown/podman-run.1.html#systemd-true-false-always), which has a default value of `true`